src/data/flexjar/useFlexjarFeedback.ts

```diff
@@ -0,0 +1,20 @@
+import { useMutation } from "@tanstack/react-query";
+import { FLEXJAR_ROOT } from "@/apiConstants";
+import { post } from "@/api/axios";
+
+export const useFlexjarFeedback = () => {
+  const path = `${FLEXJAR_ROOT}/feedback/azure`;
+  const postFlexjarFeedback = (feedback: FlexjarFeedbackDTO) =>
+    post(path, feedback);
+
+  return useMutation({
+    mutationFn: postFlexjarFeedback,
+  });
+};
+
+export interface FlexjarFeedbackDTO {
+  feedbackId: string;
+  feedback: string | null;
+  svar: string | null;
```

Comment:
Har du funnet utav hva forskjellen på disse to er? 😅

Reply:
Haha ja: `svar` er altså JA/NEI/Emoji, det de velger først. Og `feedback` er selve fritekst-tilbakemeldingen. Men enig i at det kanskje ikke er den beste DTOen 🙃
Se: 
<img width="217" alt="image" src="https://github.com/navikt/syfomodiaperson/assets/37441744/fdec4787-ffba-4b75-9d13-f563c868def1">

Reply:
Skjønner! 🙌